src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_pix2pix_zero.py

```diff
@@ -1181,16 +1198,19 @@ def invert(
                         if lambda_auto_corr > 0:
                             for _ in range(num_auto_corr_rolls):
                                 var = torch.autograd.Variable(noise_pred.detach().clone(), requires_grad=True)
-                                l_ac = self.auto_corr_loss(var, generator=generator)
+                                var_epsilon = self.get_epsilon(var, latent_model_input.detach(), t)
```

Comment:
Maybe add a one-liner comment to explain the rationale behind delegating this to the function? Might be helpful for future references.